tests/integration/organizations_test.go

```diff
@@ -103,6 +103,22 @@ func TestOrganizations(t *testing.T) {
 	assert.NotEmpty(t, newOrganization2.ID)
 	assert.Equal(t, "my-org-2", newOrganization2.Name)
 
+	// Should short  Organizations by name DESC
+	orderBy := "-name"
+	organizations, err = client.Organizations().ListAll(clerk.ListAllOrganizationsParams{
+		OrderBy: &orderBy,
```

Comment:
🔧 In a recent commit i introduced some helpers to convert string to *string for example. Let's use those to make our life easier

```suggestion
		OrderBy: stringToPtr("-name"),
```